File: projekat/src/app/entities/cars/carRentalService/car-rental-service.ts
import { Company } from '../../company/company';
import { CarRentalAgency } from '../carRentalAgency/car-rental-agency';
import { Car } from '../car/car';

export class CarRentalService extends Company {
    branches;

    /*constructor(name: string, description: string, logo: string, branches: Array<CarRentalAgency>, cars: Array<Car>){
        super(name, description, logo);

        this.branches = branches;
        this.cars = cars;
    }*/
    constructor(name: string, description: string, logo: string) { 
        super(name, description, logo);
    }
}

File: BookingService/Common/Model/CarRentalService.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Common.Model
{
    public class CarRentalService
    {
        public string Name { get; set; }
        public string Description { get; set; }
        public Address Address { get; set; }
        // Cenovnik usluga
        public List<Car> Cars { get; set; }
        public List<Address> Branches { get; set; }
    }
}
